File: C/1013 - O Maior.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int A, B, C, sum;
   
    scanf("%d %d %d", &A, &B, &C);
    
    sum = (A + B + abs(A - B)) / 2;
    sum = (C + sum + abs(sum - C)) / 2; 

    printf("%d eh o maior\n", sum);
    
    return 0;
}

File: PYTHON/1013 - O Maior.py
line = str(input(""))

split = line.split()
n1 = int(split[0]);
n2 = int(split[1]);
n3 = int(split[2]);

sum2 = ((n1 + n2) + abs(n1-n2))/2;
sum3 = ((n1 + n3) + abs(n1-n3))/2;
sum = ((sum2 + sum3) + abs(sum2-sum3))/2

print(f"{sum:.0f} eh o maior")